test/e2e/on-request-error/isr/isr.test.ts

```diff
@@ -0,0 +1,93 @@
+import { nextTestSetup } from 'e2e-utils'
+import { retry, waitFor } from 'next-test-utils'
+import { getOutputLogJson } from '../_testing/utils'
+
+const outputLogPath = 'output-log.json'
+
+describe('on-request-error - isr', () => {
+  const { next, skipped, isNextDev } = nextTestSetup({
+    files: __dirname,
+    skipDeployment: true,
+  })
+
+  if (skipped) {
+    return
+  }
+
+  if (isNextDev) {
+    it('should skip in development mode', () => {
+      // This ISR test is only applicable for production mode
+    })
+    return
+  }
```

Comment:
Why not just move it to test/development?

Reply:
it was for re-using the testing utils mostly, later I'll try reorganize it better, maybe can move to production test